content/en/news/releases/1.7.x/announcing-1.7.5/index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Announcing Istio 1.7.5
+linktitle: 1.7.5
+subtitle: Patch Release
+description: Istio 1.7.5 patch release.
+publishdate: 2020-11-19
+release: 1.7.5
+aliases:
+- /news/announcing-1.7.5
+---
+
+This release contains bug fixes to improve robustness. This release note describes what’s different between Istio 1.7.4 and Istio 1.7.5
+
+{{< relnote >}}
+
+## Changes
+
+- **Fixed** pilot agent app probe connection leak. ([Issue #27726](https://github.com/istio/istio/issues/27726))
+
+- **Fixed** how `install-cni` applies `istio-cni` plugin configuration, `Install-cni` will now remove existing `istio-cni` plugins from the CNI config before inserting `istio-cni` plugin configuration (new behavior) rather than just append a new configuration to the existing list (former behavior). ([Issue #27771](https://github.com/istio/istio/issues/27771))
+
+- **Fixed** when a node has multiple IP addresses (e.g., a VM in the mesh expansion scenario), Istio Proxy will now bind inbound listeners to the first applicable address in the list (new behavior) rather than to the last one (former behavior). ([Issue #28269](https://github.com/istio/istio/issues/28269))
+
+- **Fixed** when proxy is configured with `FILE_MOUNTED_CERTS`, gateway secret fetcher is not run.
```

Comment:
Reading this I'm not sure if this has been fixed not to run the secret fetcher when proxy is configured or if it was improperly running the secret fetcher before and now no longer is.

Reply:
Looking at https://github.com/istio/istio/pull/26978, it looks like this changes it not to run the secret fetcher with file mounted certs.

Reply:
```suggestion
- **Fixed** Istio to not run gateway secret fetcher when proxy is configured with `FILE_MOUNTED_CERTS`.
```

Reply:
Thanks! I have updated this one.